api/tests/routes/backoffice/individual_bookings_test.py

```diff
@@ -336,6 +337,32 @@ def test_list_bookings_by_status(self, authenticated_client, bookings, status, e
         rows = html_parser.extract_table_rows(response.data)
         assert set(row["Contremarque"] for row in rows) == expected_tokens
 
+    def test_list_bookings_by_cancellation_reason(self, authenticated_client):
+        bookings_factories.CancelledBookingFactory.create_batch(
+            size=3,
+            quantity=1,
+            amount=12.5,
+            token=factory.Iterator(["CNCL01", "CNCL02", "CNCL03"]),
+            stock__offer__subcategoryId=subcategories_v2.FESTIVAL_SPECTACLE.id,
+            stock__beginningDatetime=datetime.datetime.utcnow() + datetime.timedelta(days=11),
+            dateCreated=datetime.datetime.utcnow() - datetime.timedelta(days=3),
```

Comment:
[détail] Ces précisions ne me semblent pas nécessaires aux tests. On devrait pouvoir les supprimer, je pense : 

```suggestion
            token=factory.Iterator(["CNCL01", "CNCL02", "CNCL03"]),
```